Tests/test_GenBank.py

```diff
@@ -80,7 +81,8 @@ def test_write_format(self):
     def test_cleaning_features(self):
         """Test the ability to clean up feature values."""
         gb_parser = GenBank.FeatureParser(feature_cleaner=GenBank.utils.FeatureValueCleaner())
-        handle = open(os.path.join("GenBank", "arab1.gb"))
+        path = os.path.join("GenBank", "arab1.gb")
+        handle = open(path)
```

Comment:
Would indentation and a with statement to close the handle not be more elegant here (and above)?

Reply:
It doesn't really matter. The handle gets closed anyway after we leave the scope of this function.